lambda-code/nagware/lib/templates.ts

```diff
@@ -11,8 +11,14 @@ export type TemplateInfo = {
 
 export async function getTemplateInfo(formID: string): Promise<TemplateInfo> {
   try {
+    const postgresConnector =
+      await PostgresConnector.defaultUsingPostgresConnectionUrlFromAwsSecret(
+        process.env.DB_URL ?? "",
+        process.env.LOCALSTACK === "true"
```

Comment:
As long as the value is defined or falsey.  Allows devs to use True, true, 1, etc.  
```suggestion
        Boolean(process.env.LOCALSTACK)
```